Gemfile

```diff
@@ -35,6 +36,8 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug'
+  gem 'rspec-rails'
+  gem 'simplecov'
 end
 
 group :development do
```

Comment:
Including only the gems you need, awesome! :+1:  This is a good habit to have as the more dependencies you include the higher the change that your build will fail when deploying.